python/cudf/cudf/core/indexed_frame.py

```diff
@@ -3255,7 +3255,7 @@ def duplicated(
             )
             distinct = libcudf.column.Column.from_pylibcudf(plc_column)
         result = copying.scatter(
-            [cudf.Scalar(False, dtype=bool)],
+            [cudf.Scalar(False)],
```

Comment:
I think you can make this `plc.Scalar`. And then fix `scatter`'s implementation.

Reply:
Yup, I'll plan to tackle this in a follow up